twingate/internal/client/client.go

```diff
@@ -16,15 +16,19 @@ import (
 	"time"
 
 	"github.com/hashicorp/go-retryablehttp"
+	"github.com/hashicorp/go-uuid"
 	"github.com/hasura/go-graphql-client"
 )
 
+var correlationID, _ = uuid.GenerateUUID() //nolint:gochecknoglobals
```

Comment:
Why do we need a global var (and a new lint ignore) when we can just do this in `NewClient` ?
When you create a new client just send it `correlationID: uuid.GenerateUUID()`

Reply:
fixed